fetch.bs

```diff
@@ -5577,48 +5580,61 @@ these steps:
-     <a for=ReadableStream>cancel</a> <var>request</var>'s <a for=request>body</a> with undefined.
+       <li>If <var>request</var>'s <a for=request>body</a> is non-null, then
+       <a for=ReadableStream>cancel</a> <var>request</var>'s <a for=request>body</a> with undefined.
 
-     <li><p>Set <var>internalResponse</var> to <var>response</var>, if <var>response</var> is not a
-     <a>filtered response</a>; otherwise to <var>response</var>'s
-     <a for="filtered response">internal response</a>.
+       <li><p>Set <var>internalResponse</var> to <var>response</var>, if <var>response</var> is not a
+       <a>filtered response</a>; otherwise to <var>response</var>'s
+       <a for="filtered response">internal response</a>.
 
-     <li>
-      <p>If one of the following is true
+       <li>
+        <p>If one of the following is true
 
-      <ul class=brief>
-       <li><p><var>response</var>'s <a for=response>type</a> is "<code>error</code>"
+        <ul class=brief>
+         <li><p><var>response</var>'s <a for=response>type</a> is "<code>error</code>"
 
-       <li><p><var>request</var>'s <a for=request>mode</a> is "<code>same-origin</code>" and
-       <var>response</var>'s <a for=response>type</a> is "<code>cors</code>"
+         <li><p><var>request</var>'s <a for=request>mode</a> is "<code>same-origin</code>" and
+         <var>response</var>'s <a for=response>type</a> is "<code>cors</code>"
 
-       <li><p><var>request</var>'s <a for=request>mode</a> is not "<code>no-cors</code>" and
-       <var>response</var>'s <a for=response>type</a> is "<code>opaque</code>"
+         <li><p><var>request</var>'s <a for=request>mode</a> is not "<code>no-cors</code>" and
+         <var>response</var>'s <a for=response>type</a> is "<code>opaque</code>"
 
-       <li><var>request</var>'s <a for=request>redirect mode</a> is not "<code>manual</code>" and
-       <var>response</var>'s <a for=response>type</a> is "<code>opaqueredirect</code>"
+         <li><var>request</var>'s <a for=request>redirect mode</a> is not "<code>manual</code>" and
+         <var>response</var>'s <a for=response>type</a> is "<code>opaqueredirect</code>"
 
-       <li><var>request</var>'s <a for=request>redirect mode</a> is not "<code>follow</code>" and
-       <var>response</var>'s <a for=response>URL list</a> has more than one item.
-      </ul>
+         <li><var>request</var>'s <a for=request>redirect mode</a> is not "<code>follow</code>" and
+         <var>response</var>'s <a for=response>URL list</a> has more than one item.
+        </ul>
 
-      <p>then return a <a>network error</a>.
+        <p>then return a <a>network error</a>.
+      </ol>
+     <li><p>If<var>fetchResponse</var> is <a for=/>service worker timing info</a>, then:
+      <ol>
+       <li><p>Set <var>fetchParams</var>'s <a for="fetch params">timing info</a>'s
+       <a for="fetch timing info">service worker timing info</a> to
+       <var>fetchResponse</var>.
+      </ol>
     </ol>
-  </ol>
+   </ol>
```

Comment:
This seems suspect given the `<li>` indentation below. We use single-space indentation.